src/wallet/scriptpubkeyman.h

```diff
@@ -16,6 +16,8 @@
 #include <wallet/walletdb.h>
 #include <wallet/walletutil.h>
 
+#include <unordered_set>
```

Comment:
f9304b6da3237f4a5eb4ee40a149f125cfa12e1d
```suggestion
#include <unordered_map>
#include <unordered_set>
```

Reply:
Done